spec/src/main/asciidoc/graphqlapi.asciidoc

```diff
@@ -0,0 +1,57 @@
+//
+// Copyright (c) 2019 Contributors to the Eclipse Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+[[api]]
+
+==== API Annotation
+GraphQL endpoints must be annotated with the `@GraphQLApi` annotation.
+
+The `@GraphQLApi` annotation is defined in the API:
+
+.GraphQLApiAnnotation
+[source,java,numbered]
+----
+package org.eclipse.microprofile.graphql;
+
+//...
+
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+public @interface GraphQLApi {
+}
+----
+
+==== Basic POJO Example
```

Comment:
Maybe just basic example ? POJO might not be true (depending on your definition). In this can this is a CDI bean ?